### PR TITLE
Fix OS Documentation

### PR DESCRIPTION
**Pull request by bmw**
Built on #3291 and #3713.

Although I still want to see what I described in #3237 where we stop duplicating this information between our static documentation and the instruction generator, that's for another PR.

### CODE REVIEW

**Comment by bmw** [comment]
Once this merged, I'll do a (final) website update before super secret project about cryptography, pipes, and Noah the hacker is released.

**Comment by ohemorange** [comment]
Did we ever figure out what's up with the python2- vs python- thing?

**Comment by bmw** [comment]
Not really. `python-*` seems to work for some things but not others so I chose the `python2` version here. For example, this is log from Fedora 24:
```
# dnf info python-certbot-apache
Last metadata expiration check: 0:00:13 ago on Mon Nov  7 19:04:05 2016.
Error: No matching Packages to list
[root@33dcd3bf3d5a /]# dnf info python2-certbot-apache
Last metadata expiration check: 0:00:16 ago on Mon Nov  7 19:04:05 2016.
Available Packages
Name        : python2-certbot-apache
Arch        : noarch
Epoch       : 0
Version     : 0.9.3
Release     : 1.fc24
Size        : 138 k
Repo        : updates
Summary     : The apache plugin for certbot
URL         : https://pypi.python.org/pypi/certbot-apache
License     : ASL 2.0
Description : Plugin for certbot that allows for automatic configuration of apache

[root@33dcd3bf3d5a /]# dnf install python-certbot-apache
Last metadata expiration check: 0:03:03 ago on Mon Nov  7 19:04:05 2016.
Dependencies resolved.
============================================================================================================================================================================================================
 Package                                                   Arch                                     Version                                                 Repository                                 Size
============================================================================================================================================================================================================
Installing:
 augeas-libs                                               x86_64                                   1.6.0-1.fc24                                            updates                                   367 k
 certbot                                                   noarch                                   0.9.3-1.fc24                                            updates                                    26 k
 dialog                                                    x86_64                                   1.3-4.20160424.fc24                                     updates                                   226 k
 fontawesome-fonts                                         noarch                                   4.6.3-1.fc24                                            fedora                                    188 k
 fontawesome-fonts-web                                     noarch                                   4.6.3-1.fc24                                            fedora                                    385 k
 fontpackages-filesystem                                   noarch                                   1.44-17.fc24                                            fedora                                     15 k
 gc                                                        x86_64                                   7.4.2-6.fc24                                            fedora                                    104 k
 guile                                                     x86_64                                   5:2.0.13-1.fc24                                         updates                                   3.5 M
 libatomic_ops                                             x86_64                                   7.4.2-9.fc24                                            fedora                                     32 k
 libtool-ltdl                                              x86_64                                   2.4.6-12.fc24                                           updates                                    55 k
 make                                                      x86_64                                   1:4.1-5.fc24                                            fedora                                    468 k
 openssl                                                   x86_64                                   1:1.0.2h-1.fc24                                         fedora                                    499 k
 pyOpenSSL                                                 noarch                                   16.0.0-1.fc24                                           fedora                                     86 k
 python                                                    x86_64                                   2.7.12-6.fc24                                           updates                                    96 k
 python-acme-doc                                           noarch                                   0.9.3-1.fc24                                            updates                                   867 k
 python-chardet                                            noarch                                   2.2.1-5.fc24                                            fedora                                    232 k
 python-enum34                                             noarch                                   1.0.4-5.fc24                                            fedora                                     57 k
 python-idna                                               noarch                                   2.0-3.fc24                                              fedora                                     97 k
 python-ipaddress                                          noarch                                   1.0.16-2.fc24                                           fedora                                     39 k
 python-libs                                               x86_64                                   2.7.12-6.fc24                                           updates                                   5.8 M
 python-parsedatetime                                      noarch                                   1.5-3.fc24                                              fedora                                     65 k
 python-pip                                                noarch                                   8.0.2-1.fc24                                            fedora                                    1.7 M
 python-pycparser                                          noarch                                   2.14-6.fc24                                             updates                                   109 k
 python-six                                                noarch                                   1.10.0-2.fc24                                           fedora                                     34 k
 python-zope-component                                     noarch                                   4.2.2-3.fc24                                            fedora                                    117 k
 python-zope-event                                         noarch                                   4.2.0-1.fc24                                            fedora                                    178 k
 python-zope-interface                                     x86_64                                   4.1.3-4.fc24                                            fedora                                    145 k
 python2-acme                                              noarch                                   0.9.3-1.fc24                                            updates                                   171 k
 python2-augeas                                            noarch                                   0.5.0-5.fc24                                            fedora                                     30 k
 python2-certbot                                           noarch                                   0.9.3-1.fc24                                            updates                                   364 k
 python2-certbot-apache                                    noarch                                   0.9.3-1.fc24                                            updates                                   138 k
 python2-cffi                                              x86_64                                   1.5.2-1.fc24                                            fedora                                    219 k
 python2-configargparse                                    noarch                                   0.10.0-2.fc24                                           fedora                                     32 k
 python2-configobj                                         noarch                                   5.0.6-4.fc24                                            fedora                                     66 k
 python2-cryptography                                      x86_64                                   1.3.1-1.fc24                                            fedora                                    471 k
 python2-dialog                                            noarch                                   3.3.0-8.fc24                                            fedora                                     99 k
 python2-funcsigs                                          noarch                                   1.0.2-1.fc24                                            updates                                    29 k
 python2-mock                                              noarch                                   2.0.0-1.fc24                                            updates                                   130 k
 python2-ndg_httpsclient                                   noarch                                   0.4.0-3.fc24                                            fedora                                     51 k
 python2-pbr                                               noarch                                   1.8.1-4.fc24                                            fedora                                    258 k
 python2-ply                                               noarch                                   3.8-1.fc24                                              fedora                                    106 k
 python2-psutil                                            x86_64                                   3.2.1-5.fc24                                            fedora                                    130 k
 python2-pyasn1                                            noarch                                   0.1.9-6.fc24.1                                          fedora                                    105 k
 python2-pyrfc3339                                         noarch                                   1.0-3.fc24                                              fedora                                     18 k
 python2-pysocks                                           noarch                                   1.5.6-4.fc24                                            updates                                    25 k
 python2-requests                                          noarch                                   2.10.0-2.fc24                                           updates                                   107 k
 python2-setuptools                                        noarch                                   20.1.1-1.fc24                                           fedora                                    417 k
 python2-urllib3                                           noarch                                   1.15.1-3.fc24                                           updates                                   128 k
 pytz                                                      noarch                                   2016.6.1-1.fc24                                         updates                                    52 k

Transaction Summary
============================================================================================================================================================================================================
Install  49 Packages

Total download size: 18 M
Installed size: 70 M
Is this ok [y/N]:
```

File: docs/using.rst [hunk 1]
```diff
@@ -461,5 +461,3 @@ give us as much information as possible:
 - copy and paste ``certbot --version`` output
 - your operating system, including specific version
 - specify which installation method you've chosen
```
Comment (bmw) [inline docs/using.rst]: There is a final newline, there just isn't any trailing empty lines like there was before:
```
$ git checkout os-documentation 
Switched to branch 'os-documentation'
$ python -c "print open('docs/using.rst').read()[-1] == '\n'"
True
```

**Pull request merged**